test/nerves_hub_link/update_manager_test.exs

```diff
@@ -87,6 +91,51 @@ defmodule NervesHubLink.UpdateManagerTest do
     end
   end
 
+  describe "401 retry" do
+    setup do
+      port = 6543
+      devpath = "/tmp/fwup_output"
```

Comment:
Great use case for a tmp dir which should be unique to the test and you won't have to worry about cleaning up

https://hexdocs.pm/ex_unit/ExUnit.Case.html#module-tmp-dir

Reply:
Ohhhh, nice, I've switched to this, and updated other tests to use it too